package-lock.json

```diff
@@ -1,8 +1,8075 @@
 {
   "name": "overseas-entities-web",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Could you please update your local env to match [here](https://github.com/companieshouse/overseas-entities-web#requirements). Use `nvm` to change to older node/npm version, something like `nvm use 12` will do, `lockfileVersion` should be on 1

Reply:
done

Reply:
I do not know why the `package-lock.json` is broken, but could you please do the following:

- Update your env to have node `12` and npm `6` (the following command will do `nvm use 12`)
- `rm -rf node_modules/`
- copy the [package-lock.json](https://raw.githubusercontent.com/companieshouse/overseas-entities-web/main/package-lock.json) content into your package-lock.json locally on your branch
- `npm i @companieshouse/api-sdk-node` to update to the latest sdk version [2.0.47](https://github.com/companieshouse/api-sdk-node/releases/tag/2.0.47)

Note: `package-lock.json` should never be changed manually

Reply:
Yes I did copied the one on main and the previous build failed because of the companyprofile sdk version. I will do the above and see

Reply:
@mouhajer-ch steps above done

Reply:
> Yes I did copied the one on main and the previous build failed because of the companyprofile sdk version...

Yeah, sure.


Anyway, could you please pay attention, when you update a project dependency (e.g. `api-sdk-node`) you need to push the `package.json` file with the `package-lock.json`  if changed, as in this case